src/EditInfo.php

```diff
@@ -26,6 +28,11 @@ class EditInfo {
 	 * @var EditInfo::BOT|self::NOTBOT
 	 */
 	protected $bot = false;
+	
+	/**
+	 * @var integer
+	 */
+	protected $maxlag = 0;
```

Comment:
It might be better to represent no maxlag value as null.

Passing maxlag=0 to the API will do different things compared with passing no maxlag api param

Reply:
Ok, my intention was to use an "if (maxlag > 0)" before setting the query params, but this will work with null similar...

Reply:
So, it should still be possible to pass maxlag=0 to the api really

Reply:
Somewhere in the mediawiki api docs there is mentioned, that You never should use maxlag = 0 ...